src/tutorials/Mutable-File-System/08.md

```diff
@@ -0,0 +1,28 @@
+MFS allows you to move files between directories just as you would on your local machine, using the [`files.mv` method](https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/FILES.md#filesmv). It looks like this:
+
+```js
+await ipfs.files.mv(...from, to, [options])
+```
+
+`from` is the source path (or paths) of the content you'd like to move. `to` is the destination path.
+
+In order to create parent directories that don't already exist, you'll need to use the `{ parents: true }` option.
+
+You can use `files.mv` to perform a number of different operations:
+
+```js
+// move a single file into a directory
+await ipfs.files.mv('/source-file.txt', '/destination-directory')
+
+// move multiple files into a directory (note the two acceptable formats)
+await ipfs.files.mv('/source-file-1.txt', '/source-file-2.txt', '/destination-directory')
+await ipfs.files.mv(['/source-file-1.txt', '/source-file-2.txt'], '/destination-directory')
+
+// move a directory into another directory
+await ipfs.files.mv('/source-directory', '/destination-directory')
+
+// overwrite the contents of a destination file with the contents of a source file
+await ipfs.files.mv('/source-file.txt', '/destination-file.txt')
+```
+
+**Note:** `files.mv` takes longer when you move larger files. If you've been working with large files for these lessons, please click "Start Over" above the file upload window and select smaller files.
```

Comment:
Should not be true - has this been observed? It's a bug if so!

Reply:
@alanshaw I was getting no response when I submitted code for this lesson, and the thing that seemed to fix it was uploading smaller files, so I made an assumption. It's possible I misdiagnosed the problem, but it would be great if you or @fsdiogo could try completing lesson 8 with huge files and see if you can replicate the problem. Is it possible for that issue to be true in ProtoSchool but NOT on one's own locally installed IPFS for some reason?

Reply:
Here's a screenshot from the console if I try to use some .key files and .pdf files. 
![image](https://user-images.githubusercontent.com/19171465/58968279-cbbae700-8783-11e9-92d4-e502cc088964.png)

Reply:
@fsdiogo Would you be able to tell if it were the file upload stage and not the ipfs.files.mv stage causing the slowdown? Not sure if that screenshot above sheds any light.

Reply:
Just tested this and the slowdown isn't caused by the `ipfs.files.mv`, as it happens in other MFS lessons that operates with the uploaded file.

Reply:
With commit https://github.com/ProtoSchool/protoschool.github.io/pull/200/commits/f5d62e8732090f2bb9b4106ca69690280351a941 the user now gets feedback that something is in the works after submitting a lesson.